src/main/java/core/selenium/WebElementAction.java

```diff
@@ -39,6 +51,11 @@ public void clickBtn(WebElement webElement) {
         webElement.click();
     }
 
+    public void clickDropdownOption(String option) {
```

Comment:
Don't forget about the JavaDocs, and is it necessary to have create a new method? maybe you could use the clickBtn method for dropdown, and try to improve it? it's just a recommendation.

Reply:
The method to click if it is the same, what differentiates it is the wait for the type dropdown element, which does not work the same as the other methods and that is why I need its own method